deployment/docker/refresh-views.sh

```diff
@@ -1,50 +1,50 @@
 #!/usr/bin/env bash
 
-echo "$(date --iso-8601=seconds): Starting view refreshment in $DATABASE_NAME"
+echo "$(date -Iseconds): Starting view refreshment in $DATABASE_NAME"
```

Comment:
This was changed because the invocation is different now that  it's running in `alpine`, right?

Reply:
Yeah, evidently the `--iso-8601` option is not standard.  The `-I` is (supported by both GNU and non-GNU `date` commands).